acceptance/test_run_scripts/555-puppetserver-service/reboot-foss-rhel7.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
Is this script used by the tests, or is more of an example of how to run the test?

If it's more of an example, probably best to omit it from the change set, or include it as a documentation example somewhere else, ideally the same place we have the use of beaker documented as it relates to puppetserver.  This may only exist in Confluence AFAIK.